src/BlazorUI/Bit.BlazorUI/Components/ProgressIndicator/BitProgressIndicator.scss

```diff
@@ -16,6 +16,18 @@
             transform: translateX(0%);
         }
     }
+
+    @keyframes bit-pin-animation-rtl {
+        0% {
+            right: -100%;
```

Comment:
You can use `inset-inline-start` in the main animation instead of `left` property